test/Backends/LLVM.hs

```diff
@@ -0,0 +1,127 @@
+module Backends.LLVM where
+
+import Juvix.Backends.LLVM.Codegen
+import Juvix.Backends.LLVM.JIT
+import Juvix.Library
+import LLVM.AST
+import LLVM.AST.AddrSpace
+import qualified LLVM.AST.Attribute as A
+import qualified LLVM.AST.CallingConvention as CC
+import qualified LLVM.AST.Constant as C
+import qualified LLVM.AST.Global as G
+import qualified LLVM.AST.Instruction as I (function)
+import qualified LLVM.AST.Linkage as L
+import LLVM.AST.Name
+import LLVM.AST.Type
+import qualified LLVM.AST.Visibility as V
+import LLVM.Context
+import LLVM.ExecutionEngine
+import LLVM.Module
+import qualified Test.Tasty as T
+import qualified Test.Tasty.HUnit as T
+
+test_malloc_free_jit ∷ T.TestTree
+test_malloc_free_jit = T.testCase "malloc free module should jit" $ do
+  (fn ∷ Word32 → IO Word32) ← jit (Config None) mallocFreeModule "test"
+  res ← fn 7
+  43 T.@=? res
+
+mallocFreeModule ∷ LLVM.AST.Module
```

Comment:
Should probably test the functions that generate this Module as well

Reply:
Yeah - is there a way to run the `Codegen` monad?